installing/installing_vsphere/preparing-to-install-on-vsphere.adoc

```diff
@@ -6,20 +6,24 @@ include::_attributes/common-attributes.adoc[]
 
-You can also install {product-title} on vSphere by using installer-provisioned or user-provisioned infrastructure. Installer-provisioned infrastructure is ideal for installing in environments with air-gapped/restricted networks, where the installation program provisions the underlying infrastructure for the cluster. You can also install {product-title} on infrastructure that you provide. If you do not use infrastructure that the installation program provisions, you must manage and maintain the cluster resources yourself.
+== {product-title} installations on vSphere using the Assisted Installer
 
-See the xref:../../architecture/architecture-installation.adoc#installation-process_architecture-installation[Installation process] for more information about installer-provisioned and user-provisioned installation processes.
+You can install {product-title} with the link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform[{ai-full}]. This method requires no setup for the installer, and is ideal for connected environments like vSphere. Installing with the {ai-full} also provides integration with vSphere, enabling autoscaling. See xref:../../installing/installing_on_prem_assisted/installing-on-prem-assisted.adoc#installing-on-prem-assisted[Installing an on-premise cluster using the {ai-full}] for additional details.
 
-[IMPORTANT]
-====
-The steps for performing a user-provisioned infrastructure installation are provided as an example only. Installing a cluster with infrastructure you provide requires knowledge of the vSphere platform and the installation process of {product-title}. Use the user-provisioned infrastructure installation instructions as a guide; you are free to create the required resources through other methods.
-====
+== Agent-based installation of {product-title} on vSphere
+
+You can install an {product-title} cluster on vSphere using the Agent-based Installer.
+The Agent-based Installer can be used to boot an on-premises server in a disconnected environment while still taking advantage of the Assisted Installation service that is used by the {ai-full}.
+See xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#preparing-to-install-with-agent-based-installer[Preparing to install with the Agent-based Installer] for additional details.
 
 == Installer-provisioned infrastructure installation of {product-title} on vSphere
 
+You can install {product-title} on vSphere by using installer-provisioned infrastructure.
 Installer-provisioned infrastructure allows the installation program to preconfigure and automate the provisioning of resources required by {product-title}.
+Installer-provisioned infrastructure is useful for installing in environments with air-gapped/restricted networks, where the installation program provisions the underlying infrastructure for the cluster.
```

Comment:
I took this existing sentence from above, but honestly I'm not really sure it's as accurate now that the Agent-based Installer exists. So I softened "ideal" to "useful". However it's still worth checking whether or not this line is relevant anymore.